bundles/org.openhab.binding.lutron/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -135,6 +136,7 @@
 		<label>Lutron Dimmer</label>
 		<description>Controls dimmable loads</description>
 		<category>Lightbulb</category>
+		<semantic-equipment-tag>Dial</semantic-equipment-tag>
```

Comment:
LightSource ? Maybe Dial is also fine, its just that i don't match a Dial with a dimmer.